core/domain/prod_validators.py

```diff
@@ -3151,6 +3152,182 @@ def _get_model_domain_object_instance(cls, item):
             .create_community_contribution_stats_from_model(item)
         )
 
+    @classmethod
+    def _validate_translation_reviewer_counts(cls, item):
+        """For each language code, validate that the translation reviewer
+        count matches the number of translation reviewers in the datastore.
+
+        Args:
+            item: ndb.Model. CommunityContributionStatsModel to validate.
+        """
+        supported_language_codes = [
+            language_code['id'] for language_code in
+            constants.SUPPORTED_AUDIO_LANGUAGES
+        ]
+        for language_code in supported_language_codes:
+            expected_translation_reviewer_count = (
+                user_models.UserContributionRightsModel.query(
```

Comment:
Why not just fetch all the models and count the number of reviewers as per languages? [That way we will be doing one get_all (not query)]

Reply:
I didn't know you could do that - thanks Sandeep :)